### ISSUE TITLE
Mechanical power issues

### ISSUE THREAD

**Issue by tyronx** [anegostudios/VintageStory-Issues#579]
Please do report any issues related to mechanical power here. 

Here's the first by eloraam
```
One thing I noticed, though: I put two windmill blades on opposite sides of the windmill, and ran them to a common vertical shaft using a large gear for reduction.  Now the blades constantly spin up and down, like they're fighting or something.
```

**Comment by threeheadeddingo** [comment]
Any system involving attaching a large gear doesn't seem to like to "update" its connectivity - axles may not spin at all, or might spin but not spin the large gear. However, introducing a transmission and a clutch and then engaging the clutch "fixes" the gear system it's attached to and it runs beautifully.

**Comment by threeheadeddingo** [comment]
The quern never makes any progress while automated. I believe this has been an issue throughout 1.13.

**Comment by tyronx** [comment]
> 
> 
> The quern never makes any progress while automated. I believe this has been an issue throughout 1.13.

My test setup seems to work fine, can you show a screenshot of yours?

**Comment by tyronx** [comment]
We found it. Will be fixed next version. The problem can be avoided by turning the wind mill by 180 degrees

**Comment by copygirl** [comment]
From the issue linked above:
> Large Gears when put in place run hammer for a second Super fast then it all stops and even if you remove a gear here and replace or other they always end up stopping.. This is even with only helve hammer attached.. and plenty of wind windmill with just hammer of just Quern on large gear do same thing at full wind

**Comment by tyronx** [comment]
I assume all these issues are now fixed in v1.14 due to the large rewrite by radfast
